services/src/integ-test/resources/application-integ-asb.yaml

```diff
@@ -3,10 +3,10 @@ spring:
     servicebus:
       internal:
         targetAppQueue:
-          MCOL: ${MCOL.Queue:civil_sdt_inbound}
-          MCOLS: ${MCOLS.Queue:civil_sdt_inbound}
-          TEST1: ${MCOL.Queue:civil_sdt_inbound}
-      connection-string: ${SDT_QUEUE_CONNECTION_STRING:Endpoint=sb://hmc-servicebus-demo.servicebus.windows.net/;SharedAccessKeyName=Root;SharedAccessKey=wS5bXXPPip8KKEKc9A4/P6krOazMvSCwcdW4PmbsD0A=;EntityPath=civil_sdt_inbound}
+          MCOL: ${CIVIL_SDT_QUEUE:civil_sdt_inbound}
+          MCOLS: ${CIVIL_SDT_QUEUE:civil_sdt_inbound}
+          TEST1: ${CIVIL_SDT_QUEUE:civil_sdt_inbound}
```

Comment:
Should all of these point to the same queue?

Reply:
this is just an integration test and we are using only TEST1 at the moment, removed other two for now

Reply:
OK